extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/JDBCEnvironment.java

```diff
@@ -271,5 +278,24 @@ public boolean shouldUseBatchExecutor() {
         return true;
 
     }
+    
+    /**
+     * Returns a boolean value that indicates whether or not usernames should
+     * be treated as case-sensitive.
+     * 
+     * @return
+     *     true if usernames should be treated as case-sensitive, or false if
+     *     usernames should be treated as case-insensitive.
+     * 
+     * @throws GuacamoleException 
+     *     If guacamole.properties cannot be parsed.
+     */
+    public boolean getCaseSensitiveUsernames() throws GuacamoleException {
+        
+        LOGGER.warn("This database extension does not support case-sensitive username comparisons.");
```

Comment:
I wonder if we could only warn when the connection is actually configured to use case-sensitive username comparisons. 

It's not clear that it's particularly useful to warn admins about something that they're not even trying to do.

Reply:
Yeah, that's fair. I was trying to be too clever with a default implementation...